dsa-solutions/lc-solutions/0000-0099/0017-Letter Combinations of a Phone Number.md

```diff
@@ -0,0 +1,195 @@
+---
+id: Letter Combinations of a Phone Number
```

Comment:
replace `id: Letter Combinations of a Phone Number` to `id: letter-combinations-of-a-phone-number`

Reply:
Okay I will do this